front/app/api/users/types.ts

```diff
@@ -79,7 +79,7 @@ export interface IUser {
 }
 
 export interface IUserUpdate {
-  userId: string;
+  userId?: string;
```

Comment:
Just curious - is this change related to the e2e tests? Or was it needed for something else?

Reply:
One of the commands was giving me a type error because this attribute was missing so I made it optional.